client/components/SettingsPanel/settings-panel.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+    <div v-if="open" class="overlay" @click="closeOnOutsideClick">
+        <div class="settings-panel">
+            <div class="title-bar">
+                Settings <button @click="closePanel" class="close"><i class="fas fa-times"></i></button>
+            </div>
+            <div class="body">
+                <div class="menu">
+                    <button
+                        v-for="tab in tabs"
+                        @click="openSettingsTab(tab);"
+                        :key="tab.name"
+                        :class="{ active: tab === openTab }"
+                    >
+                        <i :class="tab.icon"></i> <span class="title">{{ tab.name }}</span>
+                    </button>
+                </div>
+                <div class="setting-space-wrapper">
+                    <note v-if="openTab.localChangesOnly">Changes you make here will only affect you.</note>
+                    <note v-if="!openTab.localChangesOnly" type="warning">
+                        Changes you make here will affect all connected clients.
+                    </note>
+                    <div class="setting-space"><component :is="openTab.component" /></div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import { SETTINGS_PANEL_TOGGLE } from '../../store/StaticActions';
+import PersonalizePanel from './panels/Personalize';
+import NotifyPanel from './panels/Notify';
+import StatusesPanel from './panels/Statuses';
+import Note from './Note';
+
+const tabs = [
+    {
+        icon: 'fas fa-th-list',
+        name: 'Statuses',
+        component: StatusesPanel,
+        localChangesOnly: false,
+    },
+    {
+        icon: 'fas fa-paint-brush',
+        name: 'Personalisation',
+        component: PersonalizePanel,
+        localChangesOnly: true,
+    },
+    {
+        icon: 'fas fa-bell',
+        name: 'Notifications',
+        component: NotifyPanel,
+        localChangesOnly: true,
+    },
+];
+
+export default {
+    components: { Note },
+    data() {
+        return {
+            tabs,
+            openTab: tabs[0],
+        };
+    },
+    methods: {
+        closeOnOutsideClick(event) {
+            if (event.target.className === 'overlay') {
+                this.$store.dispatch(SETTINGS_PANEL_TOGGLE);
+            }
+        },
+        closePanel() {
+            this.$store.dispatch(SETTINGS_PANEL_TOGGLE);
+        },
+        openSettingsTab(tab) {
+            this.openTab = tab;
+        },
+    },
+    computed: {
+        open() {
+            return this.$store.state.settings.settingsPanelOpen;
+        },
+    },
+};
+</script>
+
+<style lang="sass" rel="stylesheet/sass" scoped>
+$border-radius: 10px;
+$seperator-color: #F0F0F0;
```

Comment:
Don't you want to move this over to the global colors section?

Reply:
Fixed.